Perlang.sln.DotSettings

```diff
@@ -9,6 +9,10 @@
 	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ECSharpUseContinuousIndentInsideBracesMigration/@EntryIndexedValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ESettingsUpgrade_002EMigrateBlankLinesAroundFieldToBlankLinesAroundProperty/@EntryIndexedValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/UserDictionary/Words/=Callables/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/UserDictionary/Words/=getcwd/@EntryIndexedValue">True</s:Boolean>
```

Comment:
(I initially planned to have a `getcwd` which delegated to `Environment.CurrentDirectory`, but I concluded it wasn't a great idea since `getcwd` actually _exists_ on Windows as well... I am leaning towards adding a `Libc` or `System` class or something where we can put stuff like this. In a way, `Libc` would perhaps make most sense if we consider it a container for "C-like" functions like `getcwd`, perhaps `fopen` (will be needed for e.g. `Posix.mmap` I think) and so forth.)